src/hyperfine/types.rs

```diff
@@ -4,6 +4,31 @@ use std::fmt;
 /// Type alias for unit of time
 pub type Second = f64;
 
+/// Supported time units
```

Comment:
I gave it a try, let me know what you think.

I created a `units` module, and moved both `Units` and `Second` into it, as otherwise there was a circular dependency between `types::HyperfineOptions` which needs `units::Units`, but then `units::Units` needs `types::Second`.